img/atom.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
<img width="1060" alt="screenshot 2016-12-30 15 44 55" src="https://cloud.githubusercontent.com/assets/5713670/21574396/fd611e7a-cea6-11e6-90d5-dc44b49d2aaa.png">

pushed an update to match heights closer